### ISSUE TITLE
Error "env: node\r: No such file or directory" on Linux/Mac

### ISSUE THREAD

**Issue by cthulberg** [Daydreamer-riri/vite-react-ssg#22]
I'm writing to express my sincere gratitude for creating this awesome library.
However, I've come across an issue when using it on Linux/Mac systems.

## Steps to Reproduce:

1. Create a new project.
2. Run `bun add --dev vite-react-ssg` to install the package.
3. Run `bun run staticdev` to build the project.

The following error occurs:

```
env: node\r: No such file or directory
error: script "staticbuild" exited with code 127
```

## Workaround:

I found a temporary workaround by running the following command before bun run staticdev:

`find ./node_modules/vite-react-ssg/ -type f -exec dos2unix {} \\;`
This seems to fix the issue, but it's not an ideal solution.

I tried with npm and yarn and got the same error without the workaround.

**Comment by Daydreamer-riri** [comment]
![image](https://github.com/user-attachments/assets/1464b6df-c051-4ceb-b4a2-12c5b6c50935)
 Hi @cthulberg , 我这边使用 Linux 或者 WSL 都无法复现你说的问题，图片是使用 GitHub Codespace 并且正常使用 vite-react-ssg 的情况。你是否可以提供一个最小复现仓库吗？

**Comment by Daydreamer-riri** [comment]
It may possibly have something to do with the fact that I am posting the latest version on windows. I have released 0.7.0-beta.2, can you help me to try if this version works?

**Comment by cthulberg** [comment]
I was using `"vite-react-ssg": "^0.7.0-beta.1"`
Everything looks good on `"vite-react-ssg": "^0.6.4"`

Thank you!

**Comment by Daydreamer-riri** [comment]
> I was using "vite-react-ssg": "^0.7.0-beta.1"
I've also released 0.7.0-beta.2 for version 0.7.0, which can also be used if there is a need for new features (this version supports server-only loaders).
